content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -429,6 +430,58 @@ memory      0     20Gi
 pods        0     10
 ```
 
+### Cross Namespace Pod Affinity Quota
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Operators can use `CrossNamespacePodAffinity` quota scope to limit which namespaces are allowed to have pods 
+with affinity terms that cross namespaces (i.e., allowed to set `namespaces` or `namespaceSelector` fields in pod affinity terms)
```

Comment:
Is this quota scope only vaid if you have enabled the relevant feature gate?

Reply:
yes, I added a paragraph at the end.